src/execution/ast/context.cpp

```diff
@@ -32,6 +32,12 @@
 
 namespace terrier::execution::ast {
 
+Field CreatePaddingElement(uint32_t id, uint32_t size, Context *ctx) {
```

Comment:
I should've been clearer. Move this into an anonymous namespace **within** `terrier::execution::ast` like [this](https://github.com/cmu-db/noisepage/blob/master/src/execution/ast/type_printer.cpp#L11). Also, move this right above `StructType::Get()`